### ISSUE TITLE
Adding <label> elements to the rule that reverts the 'cursor' property

### ISSUE THREAD

**Issue by mesr** [elad2412/the-new-css-reset#63]
May I suggest adding *label* to the selector list of the rule the reverts the *cursor* property (line 23)? I tend to believe that CSS authors are much likelier to want the arrow cursor than the text caret for their labels.

**Comment by graymatter00** [comment]
I found an interesting article and comments about the issue of cursor-over-text in browsers.  See [https://www.impressivewebs.com/cursor-over-text-behaviour-browsers/](https://www.impressivewebs.com/cursor-over-text-behaviour-browsers/).

It seems there are differing opinions on the matter.

For me, I think it would depend on the type of website/webapp, the intended usage, and the targeted users.  Therefore, I wouldn't place a cursor setting for `label` or any other text-filled elelment in the reset.  I'd suggest applying the setting to the cursor on an as-needed basis in the site/app's custom CSS.

**Comment by elad2412** [comment]
Hi @graymatter00,

I can understand the logic of what you are saying, but still, I think everyone needs to decide for themselves in those cases.

From my perspective, I will prefer the hand "clicking" cursor because ```<label>``` is connected to something clickable like ```<input>```.

In the projects that I'm working on, there is another reset custom layer that adds things that are more my preferable, for example:

```css
button {
  cursor: pointer;
}
```

**Comment by mesr** [comment]
I agree with you guys (although I — at least partially — disagree with the opinion expressed in the article mentioned by @graymatter00 about the purpose of the text caret cursor). I myself prefer the hand pointer for `<label>` elements. I was only under the impression that the arrow cursor set by the browser's default stylesheet provided a more sensible default. This anyways is a very small detail. Thank you for putting together and maintaining this very useful piece of software!

**Comment by oinochoe** [comment]
I agree with elad2412 .
Sometimes people may want to specify "cursor" as "pointer" for the <label> tag. 
However, many people prefer to use "text" to indicate that the text is the area being scraped. 
Therefore, I think it is a part that the user should be able to select.

**Comment by mesr** [comment]
Would setting `cursor: revert` on *label* elements in the *reset* stylesheet prevent users from overriding that property in their own code? My understanding is that, unless they don't implement layers and include the *reset* stylesheet after their own stylesheet (which, as far as I understand, is not recommended anyways), there is nothing either way that prevents them from overriding the property.

**Comment by graymatter00** [comment]
@mesr
> Would setting `cursor: revert` on _label_ elements in the _reset_ stylesheet prevent users from overriding that property in their own code?

No, it would not prevent the property from being overridden by a subsequent setting of the property.

I'm not really understanding the rest of your question/comment.  However, I think a better understanding of CSS would benefit you.  I recommend [Learn to style HTML using CSS](https://developer.mozilla.org/en-US/docs/Learn/CSS) as a good place to start.

**Comment by mesr** [comment]
@graymatter00: your misunderstanding of my comment may be due to ambiguous wording on my part, but I doubt very much that it stems from a my misunderstanding of CSS.

I was merely replying to @oinochoe who seemed to suggest that setting `cursor: revert` on `<label>` elements would prevent user of this *reset* stylesheet from overriding that property. My position was that, as far as I know, if they do not implement [CSS layers](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer) **and** do include this *reset* stylesheet after their own stylesheet, then yes, this may prevent overriding some properties in some cases, but that doing so would not be a recommended CSS coding practice anyways.

As for now, I think that I will walk out of this conversation since the position of the maintainers is already clear, and that it has little impact on my appreciation of the usefulness of this stylesheet.

Thank you for taking the time to consider and evaluate my suggestion.

**Comment by oinochoe** [comment]
@mesr 
I agree. It can be overwritten by the user, but it seems that existence is necessary as a layer that needs to be processed before that. However, the question of whether it should be a cursor still remains.
